src/store/preferences/index.ts

```diff
@@ -10,8 +11,25 @@ export default {
     swimlaneStates: jsonstore.get("swimlaneStates", {}),
     search: "",
     taskEditorSectionIdAndTask: null,
+    dateFormatString: "yyyy-MM-dd"
   } as State,
-  getters: {},
+  getters: {
+    isSaveDisabled: (state: State) => (date: string) => {
+      if (date === "Invalid Date") {
+        return true;
+      }
+      return false;
+    },
+    formattedDate: (state: State) => (date: Date | undefined) => {
```

Comment:
I agree this is a solution to the comment I left previously. But I think this should just be a standard function in a date.ts file (or other name). The store is meant to be for shared state and accessing and changing that state. In this case, this function (as well as the dateformat string in the state) aren't really _state_ things. So I think it is best to pull them out into a separate place (e.g. the dateformat is just a const)